custom_components/alexa_media/__init__.py

```diff
@@ -1345,19 +1355,23 @@ async def update_listener(hass, config_entry):
     for key, old_value in hass.data[DATA_ALEXAMEDIA]["accounts"][email][
         "options"
     ].items():
-        new_value = config_entry.options.get(key)
+        new_value = config_entry.data.get(key)
         if new_value is not None and new_value != old_value:
             hass.data[DATA_ALEXAMEDIA]["accounts"][email]["options"][key] = new_value
             _LOGGER.debug(
-                "Changing option %s from %s to %s",
+                "Option %s changed from %s to %s",
                 key,
                 old_value,
                 hass.data[DATA_ALEXAMEDIA]["accounts"][email]["options"][key],
             )
-            if key == CONF_EXTENDED_ENTITY_DISCOVERY:
-                reload_needed = True
+            #            if key == CONF_EXTENDED_ENTITY_DISCOVERY:
```

Comment:
Don't leave dead code in. if you don't think we need it, just remove it.

Reply:
Sorry, I missed that one after debugging/testing.